internal/retry/retryable_errors.go

```diff
@@ -18,38 +17,49 @@ import (
 var (
 	_ basetypes.ObjectTypable  = RetryType{}
 	_ basetypes.ObjectValuable = RetryValue{}
+
+	defaultRetryableStatusCodes                = []int{429}
+	defaultRetryableReadAfterCreateStatusCodes = []int{404, 403}
```

Comment:
I think 403 retry in read-after-create only applied to some of the resources, it might not be a common behavior. So maybe use 404 as the default retry status codes for read-after-create?

Reply:
My logic here was thus: Since we have successfully created the resource (received a HTTP 200), then we should never receive a 403 or 403 when we try to GET the resource. Hence this suggestion.

Given that now there is no way to specifically alter the behaviour of the this request, then I think this is a sensible decision. WDYT?